Components/Buttons/buttons.css

```diff
@@ -0,0 +1,265 @@
+.btn {
+    cursor: pointer;
+    display: inline-block;
+    font-weight: 400;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: middle;
+    border: 1px solid transparent;
+    padding: .375rem .75rem;
+    font-size: 1rem;
+    line-height: 1.5;
+    border-radius: .25rem;
+    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;
+}
+
+.btn-primary {
+    background-color: #0069d9;
+    border-color: #0062cc;
+}
+
+.btn-primary:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #0a6dd8;
+    border-color: #0062cc;
+}
+
+.btn-secondary {
+    color: var(--bg-whitesmoke);
+    background-color: #5a6268;
+    border-color: #545b62;
+}
+
+.btn-secondary:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #6b7379;
+    border-color: #545b62;
+}
+
+.btn-success {
+    color: var(--bg-whitesmoke);
+    background-color: #218838;
+    border-color: #1e7e34;
+}
+
+.btn-success:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #0aeb3a;
+    border-color: #149432;
+}
+
+.btn-danger {
+    color: var(--bg-whitesmoke);
+    background-color: #c82333;
+    border-color: #bd2130;
+}
+
+.btn-danger:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #db172b;
+    border-color: #bd2130;
+}
+
+.btn-warning {
+    color: var(--bg-whitesmoke);
+    background-color: #e0a800;
+    border-color: #d39e00;
+}
+
+.btn-warning:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #e0b534;
+    border-color: #d39e00;
+}
+
+.btn-info {
+    color: var(--bg-whitesmoke);
+    background-color: #138496;
+    border-color: #117a8b;
+}
+
+.btn-info:hover {
+    color: var(--bg-whitesmoke);
+    background-color: #09a6be;
+    border-color: #117a8b;
+}
+
+.btn-light {
+    color: #212529;
```

Comment:
you can also add the color to a css variable, making it easier if you wish to make changes in the future